tests/Functional/FunctionalTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Functional\Tests;
+
+use Functional\Functional;
+use PHPUnit\Framework\TestCase;
+
+class FunctionalTest extends TestCase
```

Comment:
Let’s also add a test case where we check if all functions have a constant defined.

Reply:
Sorry for the late response. You can explicitly filter out the functions in the test.

Reply:
Wait, thinking about it a bit more: I would prefer to even have aliases for these two functions. They are useful with partial application.

Reply:
Well since it was originally my idea, I wanted to contribute as well, so at least I've created a PR for testcase you were talking about. https://github.com/3amprogrammer/functional-php/pull/1 it goes to the branch which is a source for this PR. I hope it helps a bit :)

Great work @3amprogrammer btw.

Reply:
@MortalFlesh can you open your PR here (you opened it for your own repository)?

Reply:
@lstrojny I will open PR here. Previous one was into this branch, so it would be merged as whole.